.github/workflows/run-tests.yml

```diff
@@ -116,12 +116,14 @@ jobs:
         run: git fetch origin ${{ github.event.pull_request.base.ref }}:${{ github.event.pull_request.base.ref }}
```

Comment:
this also won't work unless it's a PR event, right?
also, why not combine these three steps into one big script, since they're all working together (and then you just need one "if"